menus/win32.cson

```diff
@@ -16,6 +16,7 @@
       { label: 'Keymap…', command: 'application:open-your-keymap' }
       { label: 'Snippets…', command: 'application:open-your-snippets' }
       { label: 'Stylesheet…', command: 'application:open-your-stylesheet' }
```

Comment:
Hmm.. yeah.. Was thinking the same, but it's also less confusing if label and file name match:

Label | File
--- | ---
Stylesheet | `styles.less`
Syntax Stylesheet | `syntax-styles.less`

Changing `styles.less` to `ui-styles.less` would make sense but that's probably a bigger effort and maybe cause even more confusion.